example/pubspec.yaml

```diff
@@ -0,0 +1,75 @@
+name: example
+description: A new Flutter project.
+
+# The following defines the version and build number for your application.
+# A version number is three numbers separated by dots, like 1.2.43
+# followed by an optional build number separated by a +.
+# Both the version and the builder number may be overridden in flutter
+# build by specifying --build-name and --build-number, respectively.
+# In Android, build-name is used as versionName while build-number used as versionCode.
+# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
+# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
+# Read more about iOS versioning at
+# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.1.0 <3.0.0"
```

Comment:
Shouldn't we use 2.2.2? :)

Reply:
Most of the times this is done so that even peopke with outdated sdk can access the example without the need to upgarde their dart installation, at the same time that we are not obliged to update the sdk each time that dart is updated (hence the <3.0.0).

If you want i can either change the lower boundary or use the specific version, it's your call!

Reply:
Ok, agree, thank you for your effort :)